PublishedModules/glueckself/mpdclient/dialogTemplate/de.json

```diff
@@ -0,0 +1,58 @@
+{
+	"module": "mpdclient",
+	"icon": "music",
+	"description": "Control an MPD server",
+	"slotTypes": [],
+	"intents": [
+		{
+			"name": "Play",
+			"description": "User wants to start the current playlist.",
+			"enabledByDefault": false,
```

Comment:
Your intents are all disabled by default? This means they will never trigger, is that intended?

Reply:
Yes, this is intended because it they should be enabled depending on the mpd state.
When the client is disconnected, none of the intents make sense (maybe only a try-to-connect-intent, but I prefer that the module keeps trying to connect itself).

Reply:
Ok, I see, good thing! When connected, you can use MqttManager.configureIntents() (alpha2)

Reply:
Thanks! :)